tests/crypto/test_bytestring_types.py

```diff
@@ -20,30 +24,17 @@ def test_split_two_signatures():
 
 def test_split_signature_from_arbitrary_bytes():
     how_many_bytes = 10
-    signature = Signature(secure_random(65))
+    signature = Signature.from_bytes(secure_random(64))
```

Comment:
This will fail in the future when we verify that `r` and `s` are within the order of the curve.

Reply:
Only with probability 3.7e-39, but yes.

Reply:
Yeah, I mean, there are many reasons why I don't like this literal here.  So you needn't sell me on that.  However, I'm really just trying (to the degree this is possible) to refrain from adding enhancements here and just keep doing what we were doing, but with pyumbral.  Maybe make a new issue for this?